test/e2e/network/service.go

```diff
@@ -2027,6 +2027,225 @@ var _ = common.SIGDescribe("Services", func() {
 		}
 	})
 
+	ginkgo.It("should respect internalTrafficPolicy=Local when there are endpoints [Feature:ServiceInternalTrafficPolicy]", func() {
```

Comment:
Linux only tests use to have the tag `[LinuxOnly]`

Reply:
IMO I think we should avoid [LinuxOnly] here because the intent is to implement this for Windows eventually

Reply:
then add a comment please explaining that is a temporal skip until windows implements it